app/src/renderer/components/transactions/TmLiTransaction.vue

```diff
@@ -47,10 +47,7 @@ export default {
   },
   computed: {
     date() {
-      const time = moment(this.time)
-      return time.format(
-        `${moment().isSame(time, `day`) ? `` : `YYYY/MM/DD `}HH:mm`
-      )
+      return moment(this.time).format(`MMM Do YYYY, HH:mm:ss`)
```

Comment:
this logic was in place to show only the hour if the transaction happened today, and the whole date if it happened earlier, are we sure we want to drop this?

Reply:
ahhhhhh i see. no, not my intention - will revert.